app/views/prison/visits/_book_to_nomis_opt_in.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="panel" id="nomis-opt-out">
+  <% if book_to_nomis_config.possible_to_book? %>
+    <%= content_tag :p, t(".book_to_nomis_opted_in_by_default") %>
+    <%= hidden_field_tag :book_to_nomis_opted_in, true %>
+    <label class="block-label push-bottom">
+      <%= check_box_tag :book_to_nomis_opted_in, false, !book_to_nomis_config.opted_in? %>
+      <%= t('.dont_book_to_nomis') %>
+    </label>
+  <% else %>
+    <% if book_to_nomis_config.already_booked_in_nomis? %>
+      <%= content_tag :p, t(".already_booked_in_nomis") %>
+    <% else %>
+      <%= content_tag :p, t(".book_to_nomis_not_possible") %>
+    <% end %>
+  <% end %>
+
+  <div id="rejection-message-opposite">
```

Comment:
@chrishanes this does work as we expect.

Currently if you visit is set to be saved to Nomis when you select a time and then match the visitors these fields appear. Also using the checkbox does not trigger any changes.

The way I thought it was going to work was:
  - when the visit is approved, and the visit is set to be saved in nomis then staff wont see these fields.

  - when the visit is approved, and staff opt out from saving to nomis, or the visit can't be saved to nomis then staff will see these 2 fields

  - prisons that don't have book to nomis feature enabled will see no changes